src/main/resources/Task 1 - Shoe Object Modelling.txt

```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
+- create a class diagram for shoes object hierarchy.
+- implement class diagram in Java code.
+- use static code analysis tools to verify code for issues and potential bugs.
+- create a GitHub branch and Pull Request and fix all threads from code review.
```

Comment:
on diagram arrange shoe objects as a vertical tree, like Christmas tree instead spread around the page.

Reply:
The new diagram was drawn.